src/gui/pointcloud/qgspointcloudrendererpropertieswidget.cpp

```diff
@@ -191,6 +224,17 @@ void QgsPointCloudRendererPropertiesWidget::syncToLayer( QgsMapLayer *layer )
     {
       mLabels->setChecked( mLayer->renderer()->showLabels() );
       mLabelOptions->setTextFormat( mLayer->renderer()->labelTextFormat() );
+      mZoomOutOptions->setCurrentIndex( static_cast<int>( mLayer->renderer()->zoomOutBehavior() ) );
+      switch ( mLayer->renderer()->zoomOutBehavior() )
+      {
+        case Qgis::PointCloudZoomOutRenderBehavior::RenderOverview:
+          mLabels->setEnabled( false );
+          mLabelOptions->setEnabled( false );
+          break;
+        default:
```

Comment:
Avoid default when switching on enum values, so we get a build time warning if more enum values are added in future that need to be handled here

Reply:
This was already pointed out by @uclaros and fixed